hyperopt/suggest.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from future import standard_library
+from builtins import object
+import logging
+import os
+
+import numpy as np
+
+from .utils import coarse_utcnow
+from . import base
+from . import tpe
+from . import hp
+
+from pandas import DataFrame
+from datetime import datetime
+
+
+class SuggestObject(object):
```

Comment:
I suggest that we remove the `SuggestObject` class. Flatten the code and put it inside the `suggest` function.

Likewise, we may not need to separate out `serial_evaluate`.

Reply:
removed